_posts/blog/2019-08-13-Ang-Li-Three-People-in-Turing.md

```diff
@@ -0,0 +1,45 @@
+categories: blog
+date: 2019-08-13
+modified:
+share: true
+author: Ang Li
+---
+
+Some amazing people I met in Turing. :)
+
+## Bo Wang
+
+Bo Wang is a visiting researcher at Turing and work with Professor Terry Lyons based at Oxford. 
+I knew him because he is one of the organizers for Data science for mental health, an interest group aiming
+to lead to suitable clinical interventions for mental health by using multi-modal data. Bo is interested in 
+developing clinically useful application for mental health by his expert in deep learning, also he’s working 
+on some other projects. One project he’s working on is to analyze speech signals of individuals with mental 
+disorders and try to help early identify clinical high risk individuals and aid the diagnose processing in 
+psychiatry. He believe the key of such success could benefit more from novel methods integrated with prior 
+knowledge and correct data, and we could not only rely on the magic of AI. He’s very helpful and helped me 
+with establishing the azure server and applying for credits, which is crucial for my work in Turing. Bo may 
+come back to China one day and It would be exciting if I can collaborate with him on some project in the future. 
+
+## Alex Campbell
+Alex Campbell is an enrichment student at Turing and a PhD candidate at University of Cambridge. During Master 
+period, Alex used deep leanring technology in financial data mining and then he switched his main interest into
+brain science and mental disorder. Kirstie introduced Alex to me as he is also worked on projects using neuroimaging 
+data. One of his interests is to apply advanced AI models to data of human brain. Alex shared his recent work with me, 
+where he used sophisticated machine learning model on a large-sample fMRI dataset to derive pathological brain circuits 
+in autism. Alex is very helpful and really an expert at deep learnig models. He suggested me that I could use Siamese 
+network to index maturity in my project, which is very useful. 
+
+## Anujan Poologaindran
+
+Anujan Poologaindran was a neurosurgeon in Canada and worked on a magic surgery for treatment-resistant depression 
+patients. His work is very meaningful and helps many depressive patients to start a new life without the risk of 
+suicide. However, some of the patients tend to responde more to this surgery, which motiviated Anujan to further 
+investigate this heterogeneity and thus pursue a PhD degree at University of Cambridge. He's project here is pretty 
+cool and aims to identify some depression patients who could be most likely to respond for this surgery and find the 
+optimal surgery target for each individual, which involving neurosurgery, neuroimaging and machine learning. He also 
+believes that the curcuit of this surgery targets is a transdiagnostic biomarker, and thus could be useful to apply 
+this surgery for all patients who had a strong desire of suicide. I really love his ideas and hope he can move this 
```

Comment:
change "for" to "to", "had" to "have", I also recommend changing: "had a strong desire of suicide" to "had strong suicidal urges"